webapp/src/router.tsx

```diff
@@ -159,9 +159,13 @@ const FocalboardRouter = (props: Props): JSX.Element => {
                         <ChangePasswordPage/>
                     </FBRoute>}
 
-                <FBRoute path='/shared/:boardId?/:viewId?/:cardId?'>
+                <FBRoute path='/workspace/:workspaceId/shared/:boardId?/:viewId?/:cardId?'>
```

Comment:
This is here for backward compatibility, I guess, do we need the previous one (without the team or workspace) URL too?

Reply:
Also, you can use an array of paths in the `path` attribute, so probably makes sense to pass the array of all the valid urls that are for shared boards in one single route.

Reply:
Also, some routes later, we have a redirect from `/workspace/:workspaceId/shared/:boardId?/:viewId?/:cardId?` for this specific case, so we should fix that there, I guess.